frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1250,9 +1251,8 @@ export const funnelLogic = kea<funnelLogicType>({
         clearFunnel: ({}) => {
             actions.setFilters({ new_entity: values.filters.new_entity }, false, true)
         },
-        openPersonsModalForStep: ({ step, converted }) => {
-            // DEPRECATED
-            if (!values.isModalActive) {
+        openPersonsModalForStep: ({ step, stepIndex, converted }) => {
+            if (!values.isViewedOnDashboard) {
```

Comment:
Why would `openPersonsModalForStep` be called if we're on a dashboard though? In that case there should be just no `onClick` handler I'd say

Reply:
I think it's just there is a lot of places where it is called so it is cleaner to have it here than in the dozens of places where the click handler is. Isn't that the benefit of a logic 😅 ?